src/components/templates/Camp/components/RoomModalOngoingEvent.tsx

```diff
@@ -32,6 +32,19 @@ export const RoomModalOngoingEvent: React.FC<RoomModalOngoingEventProps> = ({
     roomEvents.length > 0 &&
     (currentEvent ? currentEvent : roomEvents[0]);
 
+  const joinRoom = useCallback(() => {
+    enterRoom();
+    window.open(getRoomUrl(room.url), "_blank", "noopener,noreferrer");
```

Comment:
We have a function in `utils/url` that already does all of this I believe. Let's use that here.